### PR TITLE
Added waiting for new entry in kubectl e2e test

### PR DESCRIPTION
**Pull request by piosz**
The test tends to fail on Jenkins e2e-gce (for example build no 3612) because after an entry is added to data-base it takes some time to replicate it from redis-master to redis-slave

### CODE REVIEW

File: test/e2e/kubectl.go [hunk 1]
```diff
@@ -111,40 +111,31 @@ var _ = Describe("kubectl", func() {
...
-		if err == nil && result == `{"message": "Updated"}` {
-			added = true
-			break
-		}
-	}
-	if !added {
-		Failf("Cannot added new entry in %v seconds", guestbookResponseTimeout.Seconds())
+	Logf("Trying to add a new entry to the guestbook.")
+	if !waitForGuestbookResponse(c, "set", "TestEntry", `{"message": "Updated"}`) {
+		Failf("Cannot added new entry in %v seconds.", guestbookResponseTimeout.Seconds())
```
Comment (jlowdermilk) [inline test/e2e/kubectl.go]: I don't think we want as long a timeout here and below as we use for the guestbook service to start up. Can we use 2 separate timeouts?
* guestbookStartupTimeout = 10min when initially waiting for the guestbook components to schedule and the frontend to serve traffic
* guestbookResponseTimeout = ~60s when waiting for responses from a running guestbook service.

Comment (piosz) [inline test/e2e/kubectl.go]: good idea

**Comment by jlowdermilk** [comment]
lgtm. good to merge after rebase/squash and green.

**Comment by piosz** [comment]
Thanks for review. Squashed.

**Pull request merged**